s3-sqs-cdk-java/README.md

```diff
@@ -0,0 +1,66 @@
+# Amazon S3 to Amazon SQS
+
+Java CDK example that Sends notifications from S3 to SQS when an object is created.
+
+Learn more about this pattern at Serverless Land Patterns: [https://serverlessland.com/patterns/s3-sqs-cdk-java](https://serverlessland.com/patterns/s3-sqs-cdk-java)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/cli.html) installed and configured
+  
+## Deployment Instructions
+
+1. Clone the project to your local working directory
+    ```
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+
+2. Change the working directory to this pattern's directory
+    ```
+    cd serverless-patterns/s3-sqs-cdk-java
+    ```
+
+3. From the command line, configure AWS CDK:
+   ```bash
+   cdk bootstrap ACCOUNT-NUMBER/REGION # e.g.
+   cdk bootstrap 1111111111/us-east-1
+   cdk bootstrap --profile test 1111111111/us-east-1
+   ```
+4. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the `lib/cdk-stack.ts` file:
+   ```bash
+   cdk deploy
+   ```
+5. Note the outputs from the CDK deployment process. These contain the resource names and/or ARNs which are used for testing.
+
+## How it works
+
+This CDK stack creates an S3 bucket, allows you to upload objects to that bucket, and will send notifications from S3 to SQS when an object is created in that bucket.
+
+## Testing
+
+1. Upload an object to the S3 bucket created by the deployment.
+    ```bash
+    aws s3 cp 'test-file-to-upload.txt'  s3://<BUCKET_NAME>
```

Comment:
As you have already provided the .txt file in code, I'd suggest replacing `'test-file-to-upload.txt'` with the actual file name `test_upload.txt` or rename `'test-file-to-upload.txt'` with may be `<TEST_FILE_TO_UPLOAD>`.